<?xml version="1.0" encoding="utf-8"?>
<launch>
	<param name="sigmuxSerial" type="string" value="/dev/ttyACM0"/>
	<node pkg="state_machine" type="pose_publisher_node" name="posePublisher" output="screen"/>
</launch>
